src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -21,8 +21,13 @@ namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
     public class DocumentDBConfiguration : IExtensionConfigProvider
     {
         internal const string AzureWebJobsDocumentDBConnectionStringName = "AzureWebJobsDocumentDBConnectionString";
+        // We separate the Trigger connection string in case of a mix of Trigger & output Attribute, otherwise it would generate an infinite loop
```

Comment:
Ok, that makes sense. We (kind of) have the same issue with other triggers/bindings (like Queue -> Queue, for example), but I think it's a little more obvious here b/c you're likely using the same Collection. I was just trying to keep the configuration a little smaller, if possible. Let me play with it and see if anything jumps out.